webapps/api/oauth2/redirectview.py

```diff
@@ -67,14 +67,12 @@ def get(self, request, *args, **kwargs):
         base_url = Configuration.get('/ovs/framework/webapps|oauth2.token_uri')
         client_id = Configuration.get('/ovs/framework/webapps|oauth2.client_id')
         client_secret = Configuration.get('/ovs/framework/webapps|oauth2.client_secret')
+        redirect_uri = 'https://{0}/api/oauth2/redirect/'.format(request.META['HTTP_HOST'])
         parameters = {'grant_type': 'authorization_code',
-                      'redirect_url': 'https://{0}/api/oauth2/redirect/'.format(System.get_my_storagerouter().ip),
-                      'client_id': client_id,
```

Comment:
Shouldn't the ```client_id``` be part of the call cfr https://tools.ietf.org/html/rfc6749#section-4.1.3?